handler/minidump_to_upload_parameters.cc

```diff
@@ -77,9 +77,16 @@ std::map<std::string, std::string> BreakpadHTTPFormParametersFromMinidump(
     InsertOrReplaceMapEntry(&parameters, "list_annotations", list_annotations);
   }
 
-  UUID client_id;
-  process_snapshot->ClientID(&client_id);
-  InsertOrReplaceMapEntry(&parameters, "guid", client_id.ToString());
+  if (parameters.count("_backtrace_internal_guid_override")) {
+    InsertOrReplaceMapEntry(&parameters,
+                            "guid",
+                            parameters.at("_backtrace_internal_guid_override"));
+    parameters.erase("_backtrace_internal_guid_override");
```

Comment:
should you erase parameters? We have multiple situations when you generate a dump without crash - so generating it multiple times should drop uuid flag, am I right?

Reply:
This happens in the upload handler, shouldn't affect uploads without crash. I should test it though.

Reply:
Multiple calls to DumpWithoutCrash work